frontend/src/metabase/components/Popover/TippyPopover.tsx

```diff
@@ -21,12 +22,13 @@ export interface ITippyPopoverProps extends TippyProps {
   disableContentSandbox?: boolean;
   lazy?: boolean;
   flip?: boolean;
-  sizeToFit?: boolean;
+  sizeToFit?: boolean | SizeToFitOptions;
   onClose?: () => void;
 }
 
 const PAGE_PADDING = 10;
 const OFFSET: [number, number] = [0, 5];
+const SIZE_TO_FIT_MIN_HEIGHT = 200;
```

Comment:
Not sure why linter doesn't warn/throw an error about this line. I don't think it's being used in this file.

Reply:
Odd, not sure why the linter doesn't catch this, but I've removed all the unused things in https://github.com/metabase/metabase/pull/22438/commits/52418c40d3bcff1c31dc8d390358fb4a38e467d4

Reply:
[I think I've seen this before (Slack)](https://metaboat.slack.com/archives/C505ZNNH4/p1651672938330139). Not sure if it's relavant.